tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md

```diff
@@ -10,19 +10,21 @@ primary_tag: products>sap-cloud-platform
 
 ## Details
 ### You will learn
-  - How to enable and configure workflow, business rules and process visibility services
+  - How to enable and configure workflow, business rules, process visibility and process flexibility capabilities
 
-Digital process automation for live processes is a family of services to automate business processes, manage decision logic and provide end-to-end visibility in your processes.
+**SAP Cloud Platform Workflow Management** is a cloud offering that helps you digitize structured processes with workflows and decision logic, flexibly extend and adapt business processes running in your packaged applications, manage process variants and decision logic, and monitor end-to-end process metrics in real time.
```

Comment:
This sentence is very long, which makes it hard to read. Can you please break it up into more sentences?